articles/automated-fake-news-detection/index.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/debug-node-devtools/
+title: Introduction to Automated Fake News Detection
+description: 
+author: 
+date: 2020-09-10T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/automated-fake-news-detection/hero.jpg
+    alt: Devtools example image Node.js debugger
+---
+Misinformation presents a huge challenge in today’s online society. As a result, there have been many attempts to identify and classify misinformation. Specifically, in social networking sites, blogs as well online newspapers.
+<!--more-->
+
+Since this is a very broad research area, in this article we’ll look at:
+· What is fake news and why you should care.
+· Manual vs automated fake news detection efforts.
+· Brief introduction to machine learning and deep learning techniques to fake news detection.
+· Bonus: BERT.
+The goal is to give you a gentle introduction to automated fake news detection. This should hopefully challenge you to join the fight.
+
+### What is Fake News
+
+Fake news refers to information content which is false, misleading or whose source cannot be verified. This content may be generated to intentionally damage reputation, deceive, to gain attention. The term rose to popularity during the 2016 US Presidential Elections. It was reported that fake news likely influenced the results of the elections.
+
+Various types of fake news include:
+· Clickbait. Often eye-catching content to capture readers at the expense of being factual.
+· Satire/parody. This type of content is considered to be fun and humorous thus considered to be entertaining. Yet, some readers may interpret the content as fact.
+· Propaganda. This is content meant to mislead and influence the reader.
+· Biased/partisan/hyper-partisan. Oftentimes this is biased political content claiming to be impartial.
+· Unreliable news. Journalists may publish news whose sources are unverified. Or without carrying out any form of fact checking.
+
+#### How Fake News Works
+
+Social media platforms are incredibly influential. The estimation of daily number of tweets is about 500 million. These platforms are ubiquitous. They are the go-to environment to share thoughts, feelings, opinions and intentions. This provides ideal conditions to distribute news with minimal guidelines and restrictions.
```

Comment:
estimation of daily number of tweets is about 500 million.

Do you have a source for this claim?

Reply:
Oh yes, I'll include it in the article ASAP https://www.internetlivestats.com/twitter-statistics/